.github/workflows/ci.yml

```diff
@@ -115,3 +115,20 @@ jobs:
           git add .
           git commit -m "[bot] auto-format files"
           git push
+
+  automerge:
+    runs-on: ubuntu-latest
+    needs: [validate, format]
+    if: |
+      github.event_name == 'pull_request' && 
+      github.repository == github.event.pull_request.head.repo.full_name &&
+      github.repository == 'coinhall/yacar' &&
```

Comment:
wouldn't the check for `github.repository == 'coinhall/yacar'` be enough (ie. no need for the first check)?